### ISSUE TITLE
Slow processing during optimization in quacc with NewtonNet

### ISSUE THREAD

**Issue by kumaranu** [Quantum-Accelerators/quacc#1139]
### Details about the quacc environment

- **quacc version**:
- **Python version**:
- **quacc settings**:

### What is the issue?

My earlier problem of having the transition state calculations running really slow for a few geometries still exists. To confirm whether this issue exists only inside quacc, I ran those calculations outside of quacc and I did not face the slowness in the transition state calculations.

I have attached some files here to hopefully give more context here:

### How can we easily reproduce the issue?

```python
# Paste your *minimal* example code here
```
prod.txt is a Python file, others are user input files and initial geometries for Sella. You can use 261 as the test case to reproduce the error in case you don't want to use all of them.

[068.txt](https://github.com/Quantum-Accelerators/quacc/files/13233711/068.txt)
[069.txt](https://github.com/Quantum-Accelerators/quacc/files/13233712/069.txt)
[082.txt](https://github.com/Quantum-Accelerators/quacc/files/13233713/082.txt)
[182.txt](https://github.com/Quantum-Accelerators/quacc/files/13233714/182.txt)
[193.txt](https://github.com/Quantum-Accelerators/quacc/files/13233715/193.txt)
[261.txt](https://github.com/Quantum-Accelerators/quacc/files/13233716/261.txt)
[prod.txt](https://github.com/Quantum-Accelerators/quacc/files/13233717/prod.txt)
[config30.txt](https://github.com/Quantum-Accelerators/quacc/files/13233718/config30.txt)

**Comment by Andrew-S-Rosen** [comment]
@kumaranu: It will help me greatly if you follow the issue report as-prescribed. It is there so I can best help you. That means providing a _minimal_ reproducible example of Python code pasted in the body of the issue report, among other details. I know this takes a minute or two, but it is necessary.

**Comment by Andrew-S-Rosen** [comment]
Please also confirm that this issue is unrelated to the filesystem. For instance, on many compute clusters, the filesystem is extremely slow unless you run your calculations from a scratch directory (which can be specified via the `SCRATCH_DIR` setting in quacc).

**Comment by Andrew-S-Rosen** [comment]
I am going to close this issue since it does not adhere to the template. Please feel free to re-open the issue if needed.

My guess is that this is not related to quacc but is related to the [filesystem](https://it.lbl.gov/resource/hpc/lawrencium/). Please make sure that you are running your calculations on the suggested directory by setting `SCRATCH_DIR` appropriately.

**Comment by kumaranu** [comment]
Hi @Andrew-S-Rosen,
I saw your comment yesterday and I was trying to make my script a minimal example to reproduce the same error. it is taking me some time to do that. One key reason is that it is working OK (but a bit slow as opposed to not finishing in 8 hours) when I am running locally. I will get back to you here after checking the filesystem things too. Sorry about the delay.

**Comment by Andrew-S-Rosen** [comment]
Thank you! That will help with tracking down the cause, if it is related to quacc.

**Comment by kumaranu** [comment]
Hi @Andrew-S-Rosen,
I figured out the problem. It was unrelated to the filesystem or QuAcc per se.

In my `qadaptor.yaml` file for the `fireworks` library, I had a `pre_rocket` set to load a different `conda` environment. This was the same `conda` environment that led to slow calculations. That was the reason I installed `quacc` in my older `conda` environment before talking to you about it.
So, the overall conclusion is that there was something wrong with my `conda` environment and not with `quacc`. Now, when I run the calculations they are proceeding _normally_ (meaning that `Sella` is not taking 20 minutes per iteration).
My apologies for wasting your time on this. I am thankful for the support.
